### ISSUE TITLE
truffle test - locks when after truffle init

### ISSUE THREAD

**Issue by 7--** [trufflesuite/truffle#2542]
![Screenshot from 2019-11-03 19-50-12](https://user-images.githubusercontent.com/4725003/68095010-eea51080-fe73-11e9-91eb-c40af07119ba.png)

Only after you truffle-config.js development to a client. There should be an error message saying to set it maybe.

**Comment by gnidan** [comment]
Could you provide your `truffle version` info?

After some discussion, it seems like restarting your computer might fix this.

The underlying problem seems like it might be related to Truffle's management the separate process it uses for ganache-core when running tests. This mechanism starts roughly [here](https://github.com/trufflesuite/truffle/blob/develop/packages/environment/environment.js#L67) in the code, for anyone that might want to try to investigate this.

Sounds like we might better reproduction steps for this, since half the team can reproduce and half cannot.

Thanks for raising!

**Comment by fainashalts** [comment]
@7-- we are closing this for issue maintenance. Please open a new issue if you are still having problems.
